test/unit/cbd-strategy.test.ts

```diff
@@ -163,14 +163,14 @@ describe('CbdTDecDecrypter', () => {
   it('serializes to a plain object', async () => {
```

Comment:
Should we change the name of this test component? 

```
  describe('CbdTDecDecrypter', () => {
```

to 

```
  describe('TresholdDecrypter', () => {
```

Reply:
:heavy_check_mark: